packages/mdc-slider/index.js

```diff
@@ -162,6 +165,44 @@ class MDCSlider extends MDCComponent {
     this.max = parseFloat(this.thumb_.getAttribute(strings.ARIA_VALUEMAX)) || this.max;
     this.step = parseFloat(this.thumb_.getAttribute(strings.DATA_STEP)) || this.step;
     this.value = origValueNow || this.value;
+    if (this.tickMarkSet_) {
+      this.setUpTickMarks();
```

Comment:
Also call `setUpTickMarks` in max/min/step setters.

Reply:
Got it!